recipe/meta.yaml

```diff
@@ -66,15 +65,15 @@ requirements:
     - hdf5 1.8.18|1.8.18.*
     - jpeg 9*
     - json-c  # [not win]
-    - kealib
+    - kealib 1.4.7
```

Comment:
You can have `>=1.4.7,<1.5`

Reply:
Thanks! I'll probably use that in master. This branch I wan to be as strict as possible b/c I don't really want to keep fixing it.